bigframes/functions/remote_function.py

```diff
@@ -1140,21 +1140,14 @@ def try_delattr(attr):
 
 def read_gbq_function(
     function_name: str,
-    session: Optional[Session] = None,
-    bigquery_client: Optional[bigquery.Client] = None,
+    *,
+    session: Session,
 ):
     """
     Read an existing BigQuery function and prepare it for use in future queries.
     """
-
-    # A BigQuery client is required to perform BQ operations
```

Comment:
Outside of system tests, there weren't any code paths that passed in `bigquery_client` but not `session`, so removing this unnecessary logic.